app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -21,7 +21,8 @@
   <%= render "govuk_publishing_components/components/fieldset", legend_text: legend_text do %>
     <div class="grid-row">
       <div class="column-two-thirds">
-        <%= render 'govuk_component/govspeak', content: @content_item.description %>
+        <p class="description"><%= render 'govuk_component/govspeak', content: @content_item.description %></p>
```

Comment:
Can you provide a link to an example page where this PR can be viewed? It should be somewhere on https://government-frontend-pr-909.herokuapp.com/ - the reason I ask is I think when you call the govspeak component it creates a div and then inserts the content you've specified inside it, which would mean you've got a paragraph containing a div here, which is invalid.

Reply:
I think [this an example page](https://government-frontend-pr-909.herokuapp.com/examples/service_sign_in/service_sign_in/choose-sign-in)? It looks like it hasn't put the govspeak div inside the p, it's put it afterwards, but that means you've got an empty p tag for no reason. You can probably just remove it?

Reply:
Thanks, yes that's the one. The create account page is pretty much all govspeak but I thought from the schema the description was just a string, might have got that wrong though. My only aim with wrapping it was to adjust the font size and put a break in between the text and the error (if the error is showing) so I was hoping there might be a component I could utilise already, like the lead paragraph in govuk elements? Then I wouldn't have to add a style sheet. Thanks for your help with this.

Reply:
Sorry, wont let me delete the above comment for some reason. Just looked at the heroku example and it's gov-speak, though it looks different when I run it locally probably just something I don't understand about how it processes govspeak. Basically we just want to make sure the text comes up at 19px and not too close the the error message. How it is on that heroku page is perfect so I'm not sure if this PR is required at all.